docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,159 @@
 Tutorial 5 - Taking it mobile: Android
 ======================================
 
-To deploy our applicaton to Android...
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. There is one additional
+pre-requisite: Java 8.
+
+Setting up Java 8
+=================
+
+To use BeeWare's Android tooling, you must install a Java 8 compiler and provide
+the path to it in the ``JAVA_HOME`` environment variable. This is because
+BeeWare's Android tooling depends on the Android SDK, and the Android SDK
+depends on the Java runtime and compiler being available. Some tools within
+the Android SDK specifically require version 8 of Java, so we stick to that
+in this documentation.
+
+Download & install Java 8
+-------------------------
+
+Visit the `AdoptOpenJDK website <https://adoptopenjdk.net/>`_ and
+download OpenJDK 8.
+
+On macOS, the default format is a *.pkg* installer. Download and execute the
+installer, accepting the defaults. In your terminal, run:
+``export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"``
+
+On Windows, the default is the *.msi* installer. Download and execute the
+installer. During the install process, look for an option labeled
+**Set JAVA_HOME variable**. Ensure it is enabled; if a red letter *X* precedes it,
+click the red letter *X* and choose **Will be installed on local hard drive**.
```

Comment:
Is it really a letter X, and not a checkmark?

Reply:
Well, it's a red X. I guess it's not a letter X, but a cross mark. I'm not sure if it's a letter X or a different cross mark.

Here's what I mean:

Red X mode:

![image](https://user-images.githubusercontent.com/25457/78568125-8073c980-77d6-11ea-9a44-2200150a9904.png)

Click that, and see the options:

![image](https://user-images.githubusercontent.com/25457/78568174-941f3000-77d6-11ea-9557-97f17312ec25.png)

Click one of the two "Yes" options:

![image](https://user-images.githubusercontent.com/25457/78568202-9da89800-77d6-11ea-8baa-622bfe172243.png)

Reply:
Plan: remove the word _letter_ and keep the rest